invenio_drafts_resources/services/records/components.py

```diff
@@ -40,12 +41,23 @@ def new_version(self, identity, draft=None, record=None):
         """New version callback."""
         draft['files'] = record['files']
 
+    def update_draft(self, identity, data=None, draft=None, record=None,
+                     errors=None):
+        """Update callback."""
+        draft_files = record.files
+        if draft_files.enabled and not draft_files.items():
+            errors.append({"field": "files", "messages": [
+                          "Missing uploaded files."]})
```

Comment:
1. The message should be translated:

```
from flask_babelex import gettext as _

_("...")
```

2. The message should be aligned with the one, that's currently in the UI.